keyboards/kbd6x/keymaps/othi/config.h

```diff
@@ -0,0 +1,9 @@
+#define ONESHOT_TAP_TOOGLE 2
```

Comment:
Make sure that you include `#pragma once` at the beginning.
```suggestion
#pragma once

  #define ONESHOT_TAP_TOOGLE 2
```

Reply:
i have a question regarding this, what does `#pragma once` do ? as last time i read the documentation about one shot i didn't really remember it being mentioned

Reply:
It's an include guard.  It makes sure that the compiler only includes the file once, per file, to prevent issues with redefinitions, or interactions with other files.

It may not always be necessary, but we want to make sure that this is uniformly enforced, as much as possible. 

You can read more about them here: 
https://en.wikipedia.org/wiki/Include_guard